desktop/webpack.config.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External Dependencies
+ */
+const path = require( 'path' );
+const { DefinePlugin } = require( 'webpack' );
+const CopyPlugin = require( 'copy-webpack-plugin' );
+
+const DEVELOPMENT = process.env.NODE_ENV === 'development';
+
+module.exports = {
+	context: path.resolve( __dirname, 'app' ),
```

Comment:
Nit: using `path.resolve` and `path.join` two lines below is confusing. I'd standardise on `path.join`

Reply:
Resolved